UnitySDK/Assets/ML-Agents/Scripts/Timer.cs

```diff
@@ -250,6 +294,7 @@ public void Reset(string name = "root")
         {
             m_Stack = new Stack<TimerNode>();
             m_RootNode = new TimerNode(name, true);
+            m_RootNode.Gauges = new Dictionary<string, GaugeNode>();
```

Comment:
nit: maybe do this in TimerNode.TimerNode if `isRoot` is true? Could also create lazily or store in TimerStack instead, if you're worried about memory.

Reply:
took your suggestion